tests/json_test.go

```diff
@@ -71,10 +71,11 @@ func TestJson(t *testing.T) {
 							resultMap map[string]interface{}
 							resultStr string
 						)
-						if err := conn.QueryRow(ctx, "SELECT (* except _tp_time) FROM test_json WHERE _tp_time > earliest_ts() LIMIT 1").Scan(&resultMap); assert.NoError(t, err) {
+						time.Sleep(3 * time.Second)
```

Comment:
Looks previous select from stream with "WHERE _tp_time > earliest_ts() LIMIT 1" is a little better as no need to write additional time.Sleep

Reply:
But if without sleep ,the select result become `"{\"_dummy\": 0}"`.I think it is due to `SerializationObject` will check if the column is finalized.If there is no exact json data,a default null value `"{\"_dummy\": 0}"` will be seted.

https://github.com/timeplus-io/proton-enterprise/blob/000ee5b5dda6428fd7333442f07ce28b591f8d19/src/DataTypes/Serializations/SerializationObject.cpp#L277

Maybe due to network or scheduling reasons, the json column has not been fully inserted successfully at the time of the select query,so we get the result `"{\"_dummy\": 0}"`.But I'm not sure, maybe I need some time to find out why.